src/aggregation/models/aggregationManager.ts

```diff
@@ -76,23 +87,25 @@ export class AggregationManager {
       .addSelect('max("polygon_part".resolution_meter)::numeric', 'minResolutionMeter') // minResolutionMeter - refers to the worst value (higher is worse)
       .addSelect('min("polygon_part".horizontal_accuracy_ce90)::numeric', 'maxHorizontalAccuracyCE90') // maxHorizontalAccuracyCE90 - refers to the best value (lower is better)
       .addSelect('max("polygon_part".horizontal_accuracy_ce90)::numeric', 'minHorizontalAccuracyCE90') // minHorizontalAccuracyCE90 - refers to the worst value (higher is worse)
-      .addSelect(`st_asgeojson(st_union("polygon_part".footprint), maxdecimaldigits => ${this.maxDecimalDigits}, options => 1)::json`, 'footprint')
+      .addSelect(
+        `st_asgeojson(st_geometryn(st_collect("footprintUnionCTE".footprint_union), 1), maxdecimaldigits => ${this.maxDecimalDigits}, options => 1)::json`,
+        'footprint'
+      )
       .addSelect((subQuery) => {
         return subQuery.select(`array_agg("sensors_sub_query".sensors_records)`).from((innerSubQuery) => {
           return innerSubQuery
-            .select(`DISTINCT unnest(string_to_array("polygon_part".sensors, '${this.arraySeparator}'))`, 'sensors_records')
+            .select(`unnest(string_to_array("polygon_part".sensors, '${this.arraySeparator}'))`, 'sensors_records')
+            .distinct(true)
             .from(polygonPartsEntityName, 'polygon_part')
             .orderBy('sensors_records', 'ASC');
         }, 'sensors_sub_query');
-      }, 'sensors');
-
-    const aggregationQuery = polygonPart.manager
-      .createQueryBuilder()
-      .addCommonTableExpression(aggregationCTE, 'aggregationCTE')
-      .select('*')
-      .addSelect(`trim(both '[]' from "aggregationCTE".footprint ->> 'bbox')`, 'productBoundingBox')
-      .from<AggregationLayerMetadata>('aggregationCTE', 'aggregationCTE');
+      }, 'sensors')
+      .addSelect(
+        `trim(both '[]' from (st_asgeojson(st_geometryn(st_collect("footprintUnionCTE".footprint_union), 1), maxdecimaldigits => ${this.maxDecimalDigits}, options => 1)::json ->> 'bbox'))`,
```

Comment:
We need to make sure the result of bbox consider 'lon-lat' 'lat-lon'

Reply:
it's lon-lat (checked for 4326)